libs/judicial-system/types/src/lib/case.ts

```diff
@@ -243,6 +243,35 @@ export interface Case {
   crimeScenes?: CrimeSceneMap
 }
 
+export type CaseListEntry = Pick<
```

Comment:
I think we should consider using the generated type. Maybe we can discuss that with the team?